src/js/media-controller.js

```diff
@@ -17,6 +17,7 @@ import { AttributeTokenList } from './utils/attribute-token-list.js';
 import { fullscreenApi } from './utils/fullscreenApi.js';
 import { constToCamel } from './utils/stringUtils.js';
 import { containsComposedNode } from './utils/element-utils.js';
+import MediaCaptionsButton from './media-captions-button.js';
```

Comment:
issue(blocking): Per last round of hotkeys, we're trying to avoid imports of components here. Either duplicate the logic in `handleClick` or pull it out into a reusable function.

Reply:
updated